Lombiq.Analyzers/Docs/AddingAnalyzers.md

```diff
@@ -47,39 +67,35 @@ For at least Visual Studio and JetBrains Rider you don't need any further setup
 }
 ```
 
-## How to add the analyzers to SDK-style projects from NuGet
-
-The recommended approach for SDK-style projects is adding .NET Analyzers as a submodule as explained above due to the increased control you have over configuration. However, if you aren't using Git, dislike submodules or prefer NuGet, you can also use the [NuGet package](https://www.nuget.org/packages/Lombiq.Analyzers/) to install it for just one project. Once you add the package to your project, all analyzers will be applied. Check for the latest version number [on NuGet](https://www.nuget.org/packages/Lombiq.Analyzers/).
-
-```csproj
-    <PackageReference Include="Lombiq.Analyzers" Version="<latest version>">
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
-```
+## How to add the analyzers to individual non-SDK-style .NET Framework projects (not solutions)
 
-The `<PrivateAssets>all</PrivateAssets>` is necessary to prevent the analyzers "leaking" into other projects that may consume yours.
+The key difference compared to .NET projects is that for non-SDK-style .NET Framework projects, you need to add Lombiq .NET Analyzers to each project.
 
-You can also add the package to all projects inside a folder at once [from a _Directory.Build.props_ file](https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build).
+1. Same as above - add the .NET-Analyzers repository as a Git submodule to your repository or add the NuGet package in the next step.
+2. Create a _Directory.Build.props_ file in every project folder you want to target, next to the _.csproj_ file with the following content, either referencing `Lombiq.Analyzers.NetFx` or `Lombiq.Analyzers.Orchard1`:
```

Comment:
If all the projects that you want analyzed have a common ancestor folder that only contains the projects you want analyzed, then you can get away with a single _Directory.Build.props_ file here too, right?